app/scripts/views/settings/change_password.js

```diff
@@ -54,12 +60,20 @@ define(function (require, exports, module) {
 
           throw err;
         });
+    },
+
+    onPasswordBlur: function () {
```

Comment:
nit: does this go before submit? both logically and alphabetically makes more sense that way.